sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/parser/SqlCreateTable.java

```diff
@@ -149,7 +149,7 @@ private void unparseColumn(SqlWriter writer, Schema.Field column) {
   private Table toTable() {
     return Table.builder()
         .type(SqlDdlNodes.getString(type))
-        .name(name.getSimple())
+        .name(name.names.get(name.names.size() - 1))
```

Comment:
What's this change for?

Reply:
before this PR tables could only have simple names. Now you can `create schema.subschema.tablename`, so this change gives you the `tablename`

Reply:
I see. Thanks!

Reply:
Good catch! I think `pair.right` is the same thing already parsed.